src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Common/LanguageServerConstants.cs

```diff
@@ -11,9 +11,9 @@ public static class LanguageServerConstants
 
         public const string CursorPlaceholderString = "__placeholder__";
 
-        public const string RazorSemanticTokensEndpoint = "_ms_/textDocument/semanticTokens";
+        public const string RazorSemanticTokensEndpoint = "textDocument/semanticTokens";
 
-        public const string RazorSemanticTokenLegendEndpoint = "_ms_/textDocument/semanticTokensLegend";
+        public const string RazorSemanticTokenLegendEndpoint = "textDocument/semanticTokensLegend";
```

Comment:
This should still be `_ms_/textDocument/semanticTokensLegend` because it's not in the spec.